### PR TITLE
Fix bugs on projects page [SCI-1396]

### PR DESCRIPTION
**Pull request by mlorb**
https://biosistemika.atlassian.net/browse/SCI-1396

### CODE REVIEW

**Comment by ZmagoD** [comment]
Edit action on comments don't work and the restore action from project archive don't work can you fix also that?

File: app/views/user_projects/_index.html.erb [hunk 1]
```diff
@@ -4,18 +4,19 @@
...
   <% else %>
-    <% @users.each_index do |i| user = @users[i].user %>
+
+    <% @users.each_with_index do |user_proj, i| %>
       <li>
         <% if i > 0 %><hr><% end %>
         <div class="row">
           <div class="col-xs-2">
-            <%= image_tag avatar_path(user, :icon_small), class: 'img-circle pull-left' %>
+            <%= image_tag avatar_path(user_proj.user, :icon_small), class: 'img-circle pull-left' %>
```
Comment (ZmagoD) [inline app/views/user_projects/_index.html.erb]: you can store user_proj.user in a variable

File: app/views/user_projects/_index_edit.html.erb [hunk 2]
```diff
@@ -3,26 +3,26 @@
...
 <% else %>
-  <% @users.each_index do |i| user = @users[i].user %>
+  <% @users.each_with_index do |user_proj, i| %>
   <li>
   <% if i > 0 %><hr><% end %>
     <div class="row">
 
       <div class="col-xs-2">
-        <%= image_tag avatar_path(user, :icon_small), class: 'img-circle pull-left' %>
+        <%= image_tag avatar_path(user_proj.user, :icon_small), class: 'img-circle pull-left' %>
```
Comment (ZmagoD) [inline app/views/user_projects/_index_edit.html.erb]: same here

File: app/views/user_projects/_index.html.erb (hunk 1 above)
Comment (mz3944) [inline app/views/user_projects/_index.html.erb]: če češ ne

File: app/views/user_projects/_index_edit.html.erb (hunk 2 above)
Comment (mz3944) [inline app/views/user_projects/_index_edit.html.erb]: same, če češ

**Comment by ZmagoD** [comment]
Good job! :ship:

**Pull request merged**